test/integration/etcd/data.go

```diff
@@ -45,7 +45,7 @@ func GetEtcdStorageDataForNamespace(namespace string) map[schema.GroupVersionRes
 			ExpectedEtcdPath: "/registry/configmaps/" + namespace + "/cm1",
 		},
 		gvr("", "v1", "services"): {
-			Stub:             `{"metadata": {"name": "service1"}, "spec": {"externalName": "service1name", "ports": [{"port": 10000, "targetPort": 11000}], "selector": {"test": "data"}}}`,
+			Stub:             `{"metadata": {"name": "service1"}, "spec": {"type": "LoadBalancer", "ports": [{"port": 10000, "targetPort": 11000}], "selector": {"test": "data"}}}`,
```

Comment:
why this change, why do you have WIP in the commit?

Reply:
There are a couple tests which interate all of the API types looking for `/status` subresources.  There's a table of inputs for spec and another table for status, which it tries to `apply` to `/status`.  This test tried to apply `status.loadBalancer.ingress` while the spec is `type: ClusterIP`.  That fails now.

"WIP" in comment because I was not 100% confident in it.